tests/v2/test_waku_rln_relay.nim

```diff
@@ -206,4 +206,15 @@ suite "Waku rln relay":
       check:
         # the public and secret keys together are 64 bytes
         generatedKeys.len == 64
-      debug "generated keys: ", generatedKeys 
\ No newline at end of file
+      debug "generated keys: ", generatedKeys 
+  test "membership Key Gen":
+    var key = membershipKeyGen()
+    var empty : array[32,byte]
+    check:
+      key.isSome
+      key.get().secretKey.len == 32
+      key.get().publicKey.len == 32
+      key.get().secretKey != empty
+      key.get().publicKey != empty
+    
+    debug "the generated membership key pair: ", key 
```

Comment:
newline (editor setting?)

Reply:
are you suggesting me to add newline?

Reply:
I think you meant to delete the newline because there was already a newline